src/main/resources/application-dev.yml

```diff
@@ -6,3 +6,7 @@ jobs:
     name: "BatchPrintRequests"
     cron: "0 */10 * ? * *" # every 10 minutes
     daily-limit: 1000
+  remove-vca-initial-retention-period-data:
+    enabled: true
+    name: "RemoveVcaInitialRetentionPeriodData"
+    cron: "0 */10 * ? * *" # every 10 minutes
```

Comment:
Every 10 minutes to look for data retention candidates? Wow, thats a bit keen isn't it? Would once a day at say 2am not suffice ?

Reply:
Doh! Ignore me! I've just seen this is the dev application.yml !!!
What a doofus I am! 🤣